lib/cmd/fh3/preview/url.js

```diff
@@ -0,0 +1,24 @@
+/* globals i18n */
+var fhreq = require("../../../utils/request");
+var fhc = require("../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('Show the App Preview URL of a client app'),
+  'examples' :
+    [{
+      cmd : 'fhc preview url --app=<app>',
+      desc : "Show the url preview of the app"
+    }],
+  'demand' : ['app'],
+  'alias' : {
+    'app': 'a',
+    0: 'app'
+  },
+  'describe' : {
+    'app' : i18n._("Unique 24 character GUID of the application")
+  },
+  'customCmd' : function(params, cb) {
+    var url = fhreq.getFeedHenryUrl() + 'box/srv/1.1/wid/' + fhc.curTarget + '/studio/' + params.app + '/container';
+    return cb(null, url);
```

Comment:
[Optional] I'm not sure about this but, when testing I got json object containing url. 
Maybe it should be just url and json format when json option is specified. 
I'm not 100 % sure if we should do it this way, but this will be aligned with the rest of the commands.